site/index.html

```diff
@@ -98,12 +98,33 @@
         const paginationFooters = Array.from(
           document.getElementsByClassName("docsify-pagination-container")
         );
-        const isBookPage =
-          document.location.pathname.toLowerCase().startsWith("/book") &&
-          document.location.pathname.length > 6;
+        const isBookPage = document.location.pathname
+          .toLowerCase()
+          .startsWith("/book");
         paginationFooters.forEach(
           (el) => (el.style.display = isBookPage ? "flex" : "none")
         );
+
+        // Don't show previous button on the book cover page.
+        const previousPaginationButtons = Array.from(
+          document.getElementsByClassName("pagination-item--previous")
+        );
+        const isBookCover =
+          isBookPage && document.location.pathname.toLowerCase().length < 7;

```

Comment:
length is a little magical here.  Is there a way to just firmly say that the book cover is path `x`?

Reply:
Added more precise matching.